### PR TITLE
Testing Ruby with RSpec - Tic Tac Toe #3

### PR DESCRIPTION
**Pull request by marcode95**
-In this milestone, I used RSpec to create a testing file for our Tic Tac Toe project.

### CODE REVIEW

File: spec/board_spec.rb [hunk 1]
```diff
@@ -1,76 +1,75 @@
...
+      game.taken_fields = [1, 5, 3, 2, 8, 4, 9, 6]
+      expect(game.board.display_board(game)).to eql([
+                                                      ' x | o | x ',
+                                                      '-----------',
+                                                      ' o | o | o ',
+                                                      '-----------',
+                                                      '   | x | x '
+                                                    ])
     end
   end
```
Comment (kobiyoyo) [inline spec/board_spec.rb]: - [x] Lets test for an empty board

**Review by kobiyoyo — CHANGES_REQUESTED** [review]
## Hi Team    
#### Great Job so far, everything works fine, but there are a few more things required. 
### General 
- [x] Let spec cover negative scenarios
- [x] Check the comment below 
### Make changes and submit another code review request.
*Happy coding* 
(Any questions Feel free to contact me on slack @kobiyoyo)

File: spec/board_spec.rb (hunk 1 above)
Comment (marcode95) [reply spec/board_spec.rb]: Implemented a test for an empty board and added several negative scenarios

**Review by elmejdki — APPROVED** [review]
# status: approved :heavy_check_mark: 
## Well done guys your project is complete

now you can merge to develop your project is approved.

## Happy Coding :tophat:

**Pull request merged**